zuul.d/jobs.yaml

```diff
@@ -213,26 +213,28 @@
     name: bionic
     description: Run a functional test against bionic
     parent: func-target
-    dependencies: &lint_and_unit
-        - osci-lint
-        - tox-py35
-        - tox-py36
-        - tox-py37
-        - tox-py38
+    dependencies:
+      - focal
     vars:
       tox_extra_args: bionic
 - job:
     name: focal
     description: Run a functional test against focal
     parent: func-target
-    dependencies: *lint_and_unit
+    dependencies: &lint_and_unit
```

Comment:
do we need `&lint_and_unit` here? it's not referenced by anything is it? but I guess it also doesn't hurt

Reply:
it is still referenced elsewhere in this file: https://github.com/openstack-charmers/zosci-config/blob/master/zuul.d/jobs.yaml#L267-L273